1-source-code/2-writing/2-gofmt-and-friends.md

```diff
@@ -302,6 +302,59 @@ type Person struct {
 }
 ```
 
+## `eg`
+
+`eg` is an example-based refactoring tool.
+To use it, you simply create a template file and then provide the paths you wish to refactor.
+`eg` is particulary useful for changing function or API usages.
+
+### Installing `eg`
+
+```bash
+$ go get golang.org/x/tools/cmd/eg
+$ ^get^build
```

Comment:
what does this mean?

Reply:
It is a command substitution hack: https://www.cyberciti.biz/faq/bash-history-repeat-substitution-command-syntax/

Basically, it looks for the first occurrence of `get` and replaces it with `build`.

__However__, I think this is basically the behavior of `go install`, right? I'll replace these two lines with that.